monolithic_integration_test/build.rs

```diff
@@ -0,0 +1,49 @@
+fn main() {
+    #[cfg(feature = "daphne")]
+    {
+        use std::{env, process::Command};
+
+        // Build & save off a container image for the test Daphne instance.
+        // Note: `docker build` has an `--output` flag which writes the output to somewhere, which
+        // may be a tarfile. But `docker build --output` only exports the image filesystem, and not
+        // any other image metadata (such as exposed ports, the entrypoint, etc), so we can't easily
+        // use it.
+        let build_output = Command::new("docker")
+            .args(["build", "--file=Dockerfile.test_daphne", "--quiet", "."])
```

Comment:
Similar to the other build script, we should tell Cargo to rerun this when the Dockerfile or any of the artifacts directory changes.